### DISCUSSION TITLE
Trying to set up MAUI image cropper

### DISCUSSION THREAD

**Discussion by PapLabros** [AvaloniaUI/Avalonia#17800] [Q&A]
I am trying to setup the **ImageCropper.Maui** package to use in my Avalonia mobile app. In order for the package to be setup the following method needs to be executed in the MainActivity (for Android):

```
 public class MainActivity : MauiAppCompatActivity
    {
        protected override void OnCreate(Bundle savedInstanceState)
        {
            new ImageCropper.Maui.Platform().Init(this);
            base.OnCreate(savedInstanceState);
        }
    }
```

Now 'this' is an instance of **Microsoft.Maui.MauiAppCompatActivity** which I can not find how to access in my AvaloniaMainActivity. Is this possible to do in Avalonia?

**Comment by thevortexcloud** [comment]
Avalonia has official integration with MAUI. I personally have never used it though so I can't really comment on it or say if it even works for this.

https://github.com/AvaloniaUI/AvaloniaMauiHybrid

**Reply by PapLabros** [reply]
Yes it has and I have already used it to setup a camera control and aside the airspace issue it works well. What I am referencing above is something that is required before using the AvaloniaMauiHybrid.
